experimenter/experimenter/experiments/changelog_utils.py

```diff
@@ -87,3 +92,66 @@ def generate_nimbus_changelog(experiment, changed_by, message, changed_on=None):
         message=message,
         changed_on=changed_on,
     )
+
+
+def get_formatted_change_object(field_name, field_diff, changelog, timestamp):
+    event_name = ChangeEventType.GENERAL.name
+    field_instance = NimbusExperiment._meta.get_field(field_name)
+    field_display_name = (
+        field_instance.verbose_name
+        if hasattr(field_instance, "verbose_name")
+        else field_name
+    )
+
+    old_value = field_diff["old_value"]
+    new_value = field_diff["new_value"]
+
+    if isinstance(
+        field_instance,
+        (models.ManyToManyField, models.OneToOneField, models.ManyToOneRel),
+    ):
+        event_name = ChangeEventType.DETAILED.name
+        if field_name in RelationalFields.NATIVE_MODELS:
```

Comment:
Can you add a comment here about what native models are and what sets them apart from the others?